builder/builder.go

```diff
@@ -250,7 +258,8 @@ func GetBuilders(dockerCli command.Cli, txn *store.Txn) ([]*Builder, error) {
 	builders := make([]*Builder, len(storeng))
 	seen := make(map[string]struct{})
 	for i, ng := range storeng {
-		b, err := New(dockerCli,
+		b, err := New(
```

Comment:
Sorry to nitpick, but why has the formatting changed? I agree here, that it should be on a newline :tada: - but I'd like to fix this in the linter/autoformatter, instead of just in one-off cases.

Reply:
Sorry, that was unintentional and just fixed! (I had to recreate buildx project in IntelliJ and it reset to "reformat entire file on save" 🙀)